modules/system/shells.nix

```diff
@@ -11,6 +11,17 @@ in
     environment.shells = mkOption {
       type = types.listOf (types.either types.shellPackage types.path);
       default = [];
+      defaultText = lib.literalExpression ''
+        [
+          "/bin/bash"
+          "/bin/csh"
+          "/bin/dash"
+          "/bin/ksh"
+          "/bin/sh"
+          "/bin/tcsh"
+          "/bin/zsh"
+        ]
+      '';
```

Comment:
This is misleading since setting this value isn’t a NOP, should just clarify the description if necessary.

Reply:
Fixed 👍